packages/go_router_builder/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 - Fixes integration tests. [#102799](https://github.com/flutter/flutter/issues/102799).
 
+## 1.0.2
```

Comment:
Drop the `next` section here. That line item doesn't need to be called out in the changelog. Then we should be good!

Reply:
Done 👌 
https://github.com/flutter/packages/pull/1635/commits/f1a6b33e3f45ab6066714ee1e61feb64e1b4148f